### DISCUSSION TITLE
Any recommendation if the NPI of the provider is unknown

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#457] [Q&A] [answered]
We have situations where the provider doesn't supply the NPI. Is it acceptable to send '0'  or make the filed blank? 

npi": [000000000]

**Comment by tdfow** [comment]
Since the NPI is required and zero is not a valid NPI, we are omitting these scenarios---as there is no NPI to report.  We have seen the situation, where the tax id is present--but is an SSN format.  We can't publish the tax id and must have the NPI.  If no NPI exists for this scenario, then it is omitted as we don't have the required data to report.

**Reply by Ginger228** [reply]
Thanks for the reply. I wonder if not including the NPI could be included in the safe harbor. @shaselton-usds   Do you have any guidance for us?

**Comment by Kvillafor** [comment]
We have similar situations on our side where we do not have the NPI and are currently just reporting '0' so we are not underreporting for amounts billed. 

@shaselton-usds Would you please let us know what is preferred for reporting in this case? Not reporting it altogether? Or reporting as 000000000 or 0?

**Reply by RLTx1391** [reply]
I agree that the use of "0" seems like the best solution for these scenarios, in my opinion.

**Comment by tdfow** [comment]
@shaselton @shaselton-usds   ---- Can we please get CMS feedback/direction on this discussion topic?

**Comment by shaselton-usds** [comment]
@tdfow @Ginger228 Interesting.

Does the contract not identify the provider (or provider group) that the negotiation is with? Trying to think how this might be possible.

**Reply by Ginger228** [reply]
@shaselton-usds - Our contracts are tied to the Tax Identification Number, the NPI is not a required element to contract with a provider (or provider group).

**Reply by tdfow** [reply]
@shaselton-usds  we have situations, where the Provider's tax id is SSN where we don't always have the NPI value to publish and we can't use the SSN in this scenario.

**Reply by Ginger228** [reply]
> @shaselton-usds we have situations, where the Provider's tax id is SSN where we don't always have the NPI value to publish and we can't use the SSN in this scenario.

We will have this situation as well.

**Reply by PCFLOPTeam** [reply]
We are also seeing this scenario that contracts are executed at the TIN, this is a very common practice to alleviate the maintenance of providers leaving/entering a practice/system, however the negotiated rates are available to the enterprise based on TIN.

**Reply by ryan-thomas-peterson** [reply]
@shaselton-usds, echoing this as well.  Additionally, type 2 NPIs may be created and destroyed over time as an organization makes new decisions on what qualifies as a subpart.  More info on subpart determination and NPI designation is [here](https://www.cms.gov/Outreach-and-Education/Medicare-Learning-Network-MLN/MLNProducts/downloads/NPI-What-You-Need-To-Know.pdf).  For this reason we typically contract at the TIN level for institutional type claims and don't attempt to maintain these type 2 NPIs.  Our preference would be to have a catch all code that basically says any valid NPI (or maybe one of two shorthands for any valid type 2 NPI and a different one for any valid type 1NPI) to use similar to the shorthand available for billing_codes.

**Reply by RLTx1391** [reply]
This appears to be an issue that multiple players in the industry have encountered, from what I have seen.

**Reply by Ginger228** [reply]
@shaselton-usds Any idea when (or if) we should expect guidance from CMS on this issue?

**Comment by Ginger228** [comment]
@shaselton-usds  Any recommendations from CMS on the missing NPI?  I would like to share with my development team how to code for the missing NPI for both the In-Network or Allowed Amount MRF. 

Which is the preferred data element? 
npi": [000000000]
npi": [0]
npi": []

**Comment by Ginger228** [comment]
@shaselton-usds @shaselton  Hi any update on this topic?  Due to timing and to ensure we are producing a good faith product- I'm directing my team to use all zeros for the missing NPIs.

**Comment by shaselton-usds** [comment, marked as answer]
@Ginger228 et al

Appreciate your patience and raising this valid issue!

We've made updates that can be found here: https://github.com/CMSgov/price-transparency-guide/pull/502 as well as on the [CMS website's technical clarification](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification).

For these types of contracts, please use a single "0" in the NPI array -- this would look like:
```json
"provider_groups": [{
  "npi": [0],
  "tin":{
    "type": "ein",
    "value": "11-1111111"
  }
}]
```

Why a single zero vs 10 zeros (the length of valid NPIs)? When it comes to validating a number/integer in JSON, multiple character value starting with 0 is not considered valid and should be considered a string. The NPI array currently requires values of type "number".

**Reply by tdfow** [reply]
@shaselton-usds  if the tax id is an SSN and there is no NPI---then is this expected?  Doesn't this scenario cause an issue if there is more than one tax id that is SSN based?   I see this scenario as being problematic if you have a SSN based tax id without an NPI scenario. 
"provider_groups": [{
  "npi": [0],
  "tin":{
    "type": "npi",
    "value": [0]
  }
}]

**Reply by Ginger228** [reply]
@tdfow  Thanks for bringing this up. If multiple providers still use their SSN and doesn't have a NPI should we include them on the report?

**Reply by shaselton-usds** [reply]
@tdfow How common is it that your organization that have contracts with an EIN that still uses their SSN where the contract is _**only made**_ with the EIN with no knowledge of the NPI? It would be interesting to see how much of your contracts are impacted in this scenario.

@Ginger228 Yes, it'll still be required. Same question for you. How many of your contracts are made at the EIN level without any knowledge of NPIs where your org does have knowledge that the EIN is actually a SSN?

**Reply by tdfow** [reply]
@shaselton-usds I don't have a count, but seems problematic if you have more than one with this combination for the same billing code--as  it would be repeating data in the provider_group array.   Is that what CMS is asking to be published?  If you have more than one provider using SSN and for a billing code shares the same rate---the expectation is to have the same data in separate arrays?  
I do know there are contracts that use their SSN, where there may be some without the NPI data.  I feel this is a small volume, but still a possible scenario.

**Reply by shaselton-usds** [reply]
@tdfow Agreed, I would feel like this is a pretty narrow edge case, but I don't want to discount it altogether.

As for implementation, I am assuming that this scenario, if it exists, would be small enough that utilizing the [NPPES API](https://npiregistry.cms.hhs.gov/) (this is the webform, but you can look at the underlying calls to see how the API works) to lookup the NPI would be the best approach to avoid the double zero reporting.  This could be a one-time call to populate NPIs in your systems or you could make it part of your file build workflow.

Otherwise, using the `provider_reference` on the root node and then using the `provider_reference_id` throughout varioius negotiated arrangements where this scenario comes into play would work.

**Reply by tdfow** [reply]
@shaselton-usds   I can see how the data can be interpreted as long as the tax id is EIN.  Even with having multiple provider_reference_ids--the 'data' is still showing the 'same' information other than an unique id.   The provider_reference_id would represent 'duplicate' data--wouldn't it?  
"provider_groups": [{
"npi": [0],
"tin":{
"type": "npi",
"value": [0]
}
}]
Can you clarify why the guidance would not consider to only publish the NPI as 0 when the Provider's tax id is 'EIN' and the Provider doesn't have an NPI rather than publishing the NPI as 0 regardless of the Provider tax id being ein or ssn based?
For the Allowed Amount MRF, is the expectation the same---where if there is no NPI, then publish as 0?
